cluster/charts/rook-ceph/values.yaml

```diff
@@ -62,6 +62,16 @@ csi:
   # (Optional) set user created priorityclassName for csi provisioner pods.
   # provisionerPriorityClassName: system-cluster-critical
 
+  # (Optional) policy to modifying a volume's ownership or permissions when the RBD PVC is being mounted.
+  # supported values are documented at https://kubernetes-csi.github.io/docs/support-fsgroup.html
```

Comment:
nit: if it's user facing (vs. developer), https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#configure-volume-permission-and-ownership-change-policy-for-pods is a better link?

Reply:
This is more of a CSI configuration that will be done/changed(rarely) by the admin. The link I added contains the different options the admin can change which is applicable for all the users who are consuming the pvc. Users won't have the option to change values.